express/code/blocks/pricing-table/pricing-table.js

```diff
@@ -220,17 +239,18 @@ export default async function init(el) {
   const ariaLabelForCheckIcon = await replaceKey('aria-label-pricing-icon-check', getConfig()) || 'yes';
   const ariaLabelForCrossIcon = await replaceKey('aria-label-pricing-icon-cross', getConfig()) || 'no';
 
-  const INCLUDE_ICON = `<span class="feat-icon check" aria-label=${ariaLabelForCheckIcon}></span>`;
-  const EXCLUDE_ICON = `<span class="feat-icon cross" aria-label=${ariaLabelForCrossIcon}></span>`;
+  const INCLUDE_ICON = `<span class="feat-icon check" aria-label=${ariaLabelForCheckIcon} role="img"></span>`;
+  const EXCLUDE_ICON = `<span class="feat-icon cross" aria-label=${ariaLabelForCrossIcon} role="img"></span>`;
```

Comment:
Do we have to add the role here? this is a decorative img?

Reply:
Never mind, I see why. It is because of the span element, needs a role. Thank you for catching this!